tests/test_infer/test_models/test_baichuan.py

```diff
@@ -39,7 +40,7 @@ def check_inference_engine(use_engine=False, prompt_template=None):
 
     if use_engine:
         inference_config = InferenceConfig(
-            max_output_len=output_len, prompt_template=prompt_template, dtype="fp32", use_cuda_kernel=True
+            max_output_len=output_len, prompt_template=prompt_template, use_cuda_kernel=False
```

Comment:
Is it related with rms norm, which has been fixed in #5611 ?

Reply:
Done